packages/scrollable-image/src/components/scrollable-image.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import ScrollHorizontal from './scroll-horizontal'
+import get from 'lodash/get'
+import { packageName } from '../constants'
+
+const _ = {
+  get,
+}
+
+const namespace = '__twreporterEmbeddedData'
+
+let id
+
+const scripts = document.getElementsByTagName('script')
+
+for (let s of scripts) {
+  if (s.getAttribute('data-name') === packageName) {
+    const dataID = s.getAttribute('data-id')
+    const node = document.getElementById(dataID)
+    if (node && node.getAttribute('data-status') === 'tbRendered') {
+      id = dataID
+    }
+  }
+}
+
+const config = _.get(window, [`${namespace}`, `${packageName}`, `${id}`])
+const data = _.get(config, 'data')
+const lazyload = _.get(config, 'lazyload', false)
+const container = document.getElementById(id)
+
+ReactDOM.render(
+  <ScrollHorizontal imgSrc={data} lazyload={lazyload} />,
+  container
+)
+
+container.removeAttribute('data-status')
```

Comment:
Two questions here 🙋‍♂ 
1. Why do we need to for loop script tags here?
2. Should we move this file into `src/build-code` or some other place? 
It does more than a React Component.

Reply:
First, my thought was to let main `<script>` to know which DOM element it should select by `id`. However, I think the way [dual-channel](https://github.com/twreporter/dual-channel/pull/15) use is better (passing `uuid` with `data` in array structure). It will be updated like that.

Second, it does more than a component indeed and will be moved into build-code folder.